bc-react/src/stores/EventStore.js

```diff
@@ -21,6 +21,26 @@ class EventStore extends EventEmitter{
     return this.events;
   }
 
+  updateEvent(attributes){
+    this.events = attributes
+    localStorage.setItem('event', attributes.event);
```

Comment:
does this work? can you actually get the whole event back or do you only get back a string that says "Object"